### PR TITLE
Add Bracket

### PR DESCRIPTION
**Pull request by evroon**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

I wasn't sure about the category, `Booking and Scheduling` comes closest, but that's more about appointment scheduling. [DOMJudge](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/software/domjudge.yml) is similar to this project (but more specific to programming contests), so I placed it in the same category.

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/bracket.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: "Bracket"
+website_url: "https://evroon.github.io/bracket"
+source_code_url: "https://github.com/evroon/bracket"
+description: "Flexible tournament system to build a tournament setup, add teams, schedule matches, keep track of scores and present ranking live to the public."
+licenses:
+  - MIT
+platforms:
+  - Python
+  - Javascript
+  - Docker
```
Comment (Ki-er) [inline software/bracket.yml:10]: > https://github.com/evroon/bracket#quickstart

```suggestion
platforms:
  - Docker
  - Nodejs
```

File: software/bracket.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+  - Miscellaneous
+depends_3rdparty: false
```
Comment (Ki-er) [inline software/bracket.yml:13]: Optional field so can be removed 
```suggestion
  - Miscellaneous
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Couple changes to be made.

File: software/bracket.yml (hunk 1 above)
Comment (evroon) [reply software/bracket.yml]: Ah apparently I misunderstood [this section](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#add-a-languageplatform), I thought the programming language the tool runs on is also part of this list, but it's only about what is needed to build/run the software I see now.

**Review by Ki-er — APPROVED** [review]
LGTM - Thanks!

**Review by nodiscc — APPROVED** [review]

**Pull request merged**